custom/icds_reports/static/js/directives/mobile-dot-link/mobile-dot-link.directive.js

```diff
@@ -0,0 +1,31 @@
+var url = hqImport('hqwebapp/js/initial_page_data').reverse;
```

Comment:
Inspired by https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/directives/dot-link/dot-link.directive.js

Reply:
Actually wait now that I say that I may be able to do this whole thing in HTML....

Reply:
yep, killed this file.